examples/docs_beta_snippets/docs_beta_snippets/guides/external-systems/databases/use-in-asset.py

```diff
@@ -24,33 +34,24 @@ def iris_dataset(iris_db: SnowflakeResource) -> None:
         write_pandas(conn, iris_df, table_name="iris_dataset")
 
 
-# An asset that uses a Snowflake resource called iris_db
-# and creates a new table from an existing table
+# A second asset that uses the `iris_db` resource
 @dg.asset(deps=[iris_dataset])
 def iris_setosa(iris_db: SnowflakeResource) -> None:
     with iris_db.get_connection() as conn:
-        conn.cursor().execute("""
+        conn.cursor().execute(
+            """
             CREATE OR REPALCE TABLE iris_setosa as (
             SELECT *
             FROM iris.iris_dataset
             WHERE species = 'Iris-setosa'
-        );""")
+        );"""
+        )
 
 
 defs = dg.Definitions(
     assets=[iris_dataset, iris_setosa],
-    resources={
-        # highlight-start
-        "iris_db": SnowflakeResource(
-            # Set the SNOWFLAKE_PASSWORD environment variables before running this code
-            password=dg.EnvVar("SNOWFLAKE_PASSWORD"),
-            # Update the following strings to match your snowflake database
-            warehouse="snowflake_warehouse",
-            account="snowflake_account",
-            user="snowflake_user",
-            database="iris_database",
-            schema="iris_schema",
-        )
-        # highlight-end
-    },
+    # highlight-start
+    # Include the resource in the `Definitions` object
+    resources={"iris_db": SnowflakeResource()},
```

Comment:
```suggestion
    resources={"iris_db": iris_db},
```

The following will fix the failed unit test. We were re-instantiating the snowflake resource instead of using the already defined iris_db.